components/src/Modal/Modal.js

```diff
@@ -0,0 +1,308 @@
+    left: theme.space.x1,
+    right: theme.space.x1,
+    display: "block",
+    margin: "0 auto",
+    borderBottom: `solid 1px ${theme.colors.lightGrey}`
+  }
+}));
+
+const ModalFooter = styled.div({
+  position: "relative",
+  padding: `${theme.space.x2} ${theme.space.x3}`,
+  backgroundColor: transparentize(0.1, theme.colors.white),
+  zIndex: 2,
+  ":after": {
+    content: "''",
+    position: "absolute",
+    top: 0,
+    left: theme.space.x1,
+    right: theme.space.x1,
+    display: "block",
+    margin: "0 auto",
+    borderBottom: `solid 1px ${theme.colors.lightGrey}`
+  }
+});
+
+const ModalCloseButton = styled(CloseButton)({
+  position: "absolute",
+  top: "12px",
+  right: theme.space.x2,
+  zIndex: 3
+});
+
+const StyledReactModal = styled(ReactModal)(
+  ({ maxWidth }) => ({
+    maxWidth
+  }),
+  {
+    ":focus": {
+      outline: "none"
+    },
+    display: "flex",
+    flexDirection: "column",
+    position: "relative",
+    top: 0,
+    left: 0,
+    right: 0,
+    bottom: 0,
+    overflow: "hidden",
+    backgroundColor: theme.colors.white,
+    borderRadius: theme.radii.medium,
+    border: null,
+    width: "100%",
+    maxHeight: "70vh",
+    margin: `0px ${theme.space.x2}`,
+    padding: 0,
+    [`@media only screen and (max-width: ${theme.breakpoints.small})`]: {
+      width: "100%",
+      maxWidth: "100%"
+    },
+    button: {
+      fontFamily: theme.fonts.base
+    },
+    "*": {
+      boxSizing: "border-box"
+    },
+    color: theme.colors.black,
+    fontFamily: theme.fonts.base,
+    fontSize: theme.fontSizes.medium,
+    lineHeight: theme.lineHeights.base,
+    WebkitFontSmoothing: "antialiased",
+    MozOsxFontSmoothing: "grayscale"
+  }
+);
+
+const overlayStyle = {
+  position: "fixed",
+  top: 0,
+  left: 0,
+  right: 0,
+  bottom: 0,
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+  backgroundColor: transparentize(0.5, theme.colors.blackBlue),
+  zIndex: theme.zIndex.overlay
+};
+
+class Modal extends React.Component {
+  getPrimaryButtonComponent() {
+    const { type } = this.props;
+
+    if (type === "informative") {
+      return PrimaryButton;
+    } else if (type === "danger") {
+      return DangerButton;
+    } else {
+      return Button;
+    }
+  }
+
+  getPrimaryButton() {
+    const { primaryButton, type } = this.props;
+    const PrimaryButtonComponent = this.getPrimaryButtonComponent(type);
+
+    return <PrimaryButtonComponent {...primaryButton}>{primaryButton.label}</PrimaryButtonComponent>;
+  }
+
+  getSecondaryButtons() {
+    const { secondaryButtons } = this.props;
+
+    if (!Array.isArray(secondaryButtons)) {
+      return null;
+    }
+
+    return secondaryButtons.map(button => (
+      <Button {...button} key={button.label}>
+        {button.label}
+      </Button>
+    ));
+  }
+
+  getModalButtons() {
+    const { buttonAlignment } = this.props;
+
+    if (buttonAlignment === "spaced") {
+      return (
+        <React.Fragment>
```

Comment:
`<>`

Reply:
Sounds good, Ive changed all instances of React.Fragment to use this syntax to be consistent